cmd/account-server/etc/account_server.yaml

```
@@ -85,6 +85,13 @@ billAllocation:
   awsCommonExpense:
     excludeAccountCloudIDs:
 #      - 123456789
+  # 根账号需要抵扣的项目列表
+  awsDeductAccountItems:
+    deductItemTypes:
+      00000001: # 根账号ID
+        line_item_line_item_type: # 项目涵盖的费用类型
+          - Tax
+          - Fee
```

考虑改下层级？这样方便给每个账号单独配置
```suggestion
  awsDeductAccountItems:
   00000001: # 根账号ID
      deductIFilter:
        line_item_line_item_type: # 项目涵盖的费用类型
          - Tax
          - Fee
```

---

可以看情况改

---

尝试改了下发现改动有点大，还得从头测一遍，本期先不改动了，下次这里有需求时可以顺手改一下结构